src/zenml/integrations/seldon/steps/seldon_deployer.py

```diff
@@ -383,3 +392,147 @@ def seldon_custom_model_deployer_step(
     )
 
     return service
+
+
+@step(enable_cache=False)
+def seldon_mlflow_registry_deployer_step(
+    params: SeldonDeployerStepParameters,
+) -> SeldonDeploymentService:
+    """Seldon Core model deployer pipeline step.
+
+    This step can be used in a pipeline to implement continuous
+    deployment for a MLflow model with Seldon Core.
+
+    Args:
+        params: parameters for the deployer step
+
+    Returns:
+        Seldon Core deployment service
+
+    Raises:
+        ValueError: if registry_model_name is not provided
+        ValueError: if neither registry_model_version nor
+            registry_model_stage is provided
+        ValueError: if the MLflow experiment tracker is not available in the
+            active stack
+        RuntimeError: if no model version is found in the MLflow model registry.
+    """
+    # import here to avoid failing the pipeline if the step is not used
+    from zenml.integrations.mlflow.experiment_trackers import (
+        MLFlowExperimentTracker,
+    )
+    from zenml.integrations.mlflow.model_registries import MLFlowModelRegistry
+
+    # check if the MLflow experiment tracker, MLflow model registry and
+    # Seldon Core model deployer are available
+    if not params.registry_model_name:
+        raise ValueError(
+            "registry_model_name must be provided to the MLflow"
+            "model registry deployer step."
+        )
+    elif not params.registry_model_version and not params.registry_model_stage:
+        raise ValueError(
+            "Either registry_model_version or registry_model_stage must"
+            "be provided in addition to registry_model_name to the MLflow"
+            "model registry deployer step. Since the"
+            "mlflow_model_registry_deployer_step is used in conjunction with"
+            "the mlflow_model_registry."
+        )
+    if params.service_config.implementation != "MLFLOW_SERVER":
+        raise ValueError(
+            "This step only allows MLFLOW_SERVER implementation with seldon"
+        )
+    # Get the active model deployer
+    model_deployer = cast(
+        SeldonModelDeployer, SeldonModelDeployer.get_active_model_deployer()
+    )
+
+    # fetch the MLflow experiment tracker
+    experiment_tracker = Client().active_stack.experiment_tracker
+    if not isinstance(experiment_tracker, MLFlowExperimentTracker):
+        raise ValueError(
+            "MLflow model deployer step requires an MLflow experiment "
+            "tracker. Please add an MLflow experiment tracker to your "
+            "stack."
+        )
```

Comment:
This should not be here anymore, if you implemented the abstraction properly, the mlflow model registry should run this check.